src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/AssemblerProvider.java

```diff
@@ -99,9 +99,9 @@ private VaultPackageAssembler lazyConstruct(@NotNull BundleSlingInitialContentEx
     private void addPathFilterSetToAssemblerFilter(@NotNull PathEntry pathEntry, @NotNull VaultPackageAssembler assembler) {
         ImportMode importMode;
         if (pathEntry.isOverwrite()) {
-            importMode = ImportMode.UPDATE;
+            importMode = ImportMode.REPLACE;
         } else {
-            importMode = ImportMode.MERGE;
+            importMode = ImportMode.MERGE_PROPERTIES;
```

Comment:
is this change absolutely required for the task at hand?
do you have a test illustrating the effect?

i am not opposed but want to avoid introducing new problems.
maybe @kwin can also provide some input to make sure we understand any potential unwanted side-effects